doc/dev/ceph_krb_auth.rst

```diff
@@ -0,0 +1,335 @@
+======================================================================
+A Detailed Documentation on How to Set up Ceph Kerberos Authentication
+======================================================================
+Daniel Oliveira
+Last update: Aug 14, 2018
+
+This document provides deeper detail on the Cephx authorization protocol whose high level flow 
```

Comment:
This file is still around, and should either be replaced with the previously mentioned "proper one", or dropped entirely.